src/main/webapp/app/exercises/modeling/assess/modeling-assessment-editor/modeling-assessment-dashboard.component.ts

```diff
@@ -116,18 +117,20 @@ export class ModelingAssessmentDashboardComponent implements OnInit, OnDestroy {
                 // the server should have filtered these submissions already
                 this.submissions = res.body!.filter((submission) => submission.submitted);
                 this.submissions.forEach((submission) => {
-                    if (submission.result) {
+                    if (getLatestSubmissionResult(submission)) {
```

Comment:
Please save the result in a global variable and change it also in the html accordingly

Reply:
This is a reoccurring thing, so check also the other parts where `getLatestSubmissionResult` is used :)